.github/CODEOWNERS

```diff
@@ -5,6 +5,4 @@
 # the repo. Unless a later match takes precedence,
 # @global-owner1 and @global-owner2 will be requested for
 # review when someone opens a pull request.
-*       @Visssarion
-*       @0x78f1935
-*       @HaiKonofanDesu
+*       @Visssarion @0x78f1935 @HaiKonofanDesu
```

Comment:
Oh so it has to be all in one line? That's dumb.

Reply:
Yeah! It appears to be that the `*` is a wildcard for the root dir of the project. So in this case, we make 3 people owner over 1 directory :). Totally my bad.